src/components/heartrate/Ppg.cpp

```diff
@@ -2,106 +2,276 @@
   SPDX-License-Identifier: LGPL-3.0-or-later
   Original work Copyright (C) 2020 Daniel Thompson
   C++ port Copyright (C) 2021 Jean-François Milants
+  Modified to use frequency space pulse rate metrics and ALS sensor monitoring Copyright (C) 2022 Ceimour
```

Comment:
I wouldn't call this modified, when there's nothing original left. This makes me wonder if this should be a new file, so it's not confused with the old one.

Reply:
Yep, this PR does not keep any of the original code of ppg.c/h. How should that be handled regarding the license and copyrights?

Reply:
There's not really any licensed code left, so maybe it isn't applicable? Unless we can come up with a new name for this file, we could remove this file in one commit and add a new file with the same name in another.

Reply:
So we could either remove the copyright notice or remove the old file and add the new file with the same name, then.

Reply:
Given that none of the original code is retained, the copyright notice should be removed in the commit that removes the last bit of the original code.

Reply:
@JF002 Licensing and copyright removed.